src/components/header/Header.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Search from 'components/search';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import styles from './styles.module.css'
+
+const Header = () => {
+    return (
+        <header className={styles.header}>
+            <Container>
+                <Row>
+                    <Col>
+                        <a className={styles.logo} href="#">
+                            <strong>netflix</strong>
+                            roulette
+                        </a>
+                    </Col>
+                    <Col className="text-right">
+                        <button className={styles.addMoreBtn}>+ Add More</button>
```

Comment:
Good case to for smart/dumb component separation - make a separate `Button` component with few props. Especially that you have `Button` component in `Search.jsx` imported from `react-bootstrap` and using different kinds of same elements (one is simple html element and another is a React component) doesn't complies well with React principles. If you wanna use simple button with custom styles and not one from bootstrap - you can make a wrapper with conditional rendering or whatever you like, but the button should be only one in this case:)